lib/src/install/baseline.rs

```diff
@@ -367,6 +381,27 @@ pub(crate) fn install_create_rootfs(
             ];
             (rootdev, Some(kargs))
         }
+        BlockSetup::PasswordLuks => {
+            let uuid = uuid::Uuid::new_v4().to_string();
+            let password = opts.luks_password.as_ref().unwrap();
+            let root_devpath = root_partition.path();
+
+            Task::new("Initializing LUKS for root", "cryptsetup")
```

Comment:
This looks reasonable, however for consistency it could make sense to use `systemd-cryptenroll` for this too.